torch/testing/_internal/common_utils.py

```diff
@@ -1723,7 +1723,8 @@ def f_retry(*args, **kwargs):
 # Methods for matrix and tensor generation
 
 def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, high=None,
-                requires_grad: bool = False, noncontiguous: bool = False) -> torch.Tensor:
+                requires_grad: bool = False, noncontiguous: bool = False,
+                exclude_values: list = []) -> torch.Tensor:
```

Comment:
Like `flake8` suggests don't use mutable structure for default argument. More context and workaround, https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Yes, done! Added `None` here.